tools/generator/exemple.csv

```diff
@@ -0,0 +1,9 @@
+experiments_name,subjects_names,years,months,days,sessions_numbers,comments
+neo,enya,2020,12,01,001,none
+neo,enya,2020,12,01,002,none
+neo,enya,2020,02,01,003,none
+neo,zamba,2020,12,12,001,none
+neo,zamba,2020,2,3,010,none
+aerial,zimba,2020,12,06,001,none
+aerial,zimba,2020,12,01,010,none
+
```

Comment:
please put some example comments! and also maybe add some spaces or tabs to make it more easily readable?

Reply:
What do you mean by "example comments!" ? And It's a CSV file, so I'm not sure if I can add some space or tabs without modifying the format ?

Reply:
the last column of this file is called "comments", but for now, you have the value "none" for all of them... especially that if you put "none", there will be "none" at the end of the name of the directory...

so it'd be good to have a few examples... for example, you can put:
- protocol-v1
- protocol-v2
- acute
- chronic
but also, you should have at least one line with an empty comment

Reply:
Ok ,but I've made a security check if there is a NULL or empty value in the CSV It aborts the generation .

Reply:
ok, let's just not worry about these details... so put something on each line of the exemple.csv file, and that will be enough! line by line, you can put this:
- protocol-v1
- protocol-v2
- protocol-v1
- vsdi
- vsdi
- ephys-acute
- ephys-chronic

Reply:
Ok done =)

Reply:
ok, good for me!